web/src/components/storage/ProposalVolumes.jsx

```diff
@@ -65,6 +65,8 @@ const AutoCalculatedHint = (volume) => {
           // TRANSLATORS: list item, this affects the computed partition size limits
           // %s is replaced by a list of the volumes (like "/home, /boot")
           <ListItem>{sprintf(_("Presence of other volumes (%s)"), sizeRelevantVolumes.join(", "))}</ListItem>}
+        {adjustByRam &&
+          <ListItem>{_("The amount of RAM in the system")}</ListItem>}
```

Comment:
I'd add some context in the `// TRANSLATORS:` comment here, it might not be obvious what does the text mean without more details , this could cause wrong or confusing translations.

Reply:
Updated and added comments